Updated bedbase ids and docs

khoroshevskyi opened the pull request:
## Changes:
- Updated readme file - updated bedbase links with correct bed files ids

khoroshevskyi committed:
Updated bedbase ids and docs

mdozmorov commented:
@khoroshevskyi, thank you, that was long overdue. Before merging, can you correct several links that return "Not found"? https://bedbase.org/bed/ and https://bedbase.org/docs/v1

khoroshevskyi committed:
Fixed missing files and incorrect links

khoroshevskyi committed:
Fixed incorrect links

khoroshevskyi commented:
@mdozmorov, I fixed incorrect links and added 2 missing files,
Additionally FYI, ['hg19.UCSC.scaffold', 'mm10.UCSC.scaffold', 'mm10.UCSC.fragment'] files are missing in google drive, and I couldn't add them to the BEDbase

mdozmorov commented:
Thank you! I'll get in touch about the missing files, we have 8 more to add.
